src/internal/packager/helm/utils.go

```diff
@@ -61,7 +62,12 @@ func (h *Helm) parseChartValues() (chartutil.Values, error) {
 	}
 
 	providers := getter.Providers{httpProvider}
-	return valueOpts.MergeValues(providers)
+	chartValues, err := valueOpts.MergeValues(providers)
+	if err != nil {
+		return chartValues, err
+	}
+
+	return helpers.MergeMapRecursive(chartValues, h.ValuesOverrides), nil
```

Comment:
what happens if `h.ValuesOverrides` is `nil`?

Reply:
it will just return